lesson_23/app.js

```diff
@@ -0,0 +1,276 @@
+//1. Prototype inheritance
+
+class bankAccount {
+  constructor(balance, accountHolder) {
+    this._balance = balance;
+    this.accountHolder = accountHolder;
+  }
+  get getBalance() {
```

Comment:
if you use a getter or setter just use name
example balance
if you use a specific function add prefix getBalance, setBalance